Sources/xcproj/CommentedString.swift

```diff
@@ -31,16 +31,15 @@ struct CommentedString {
         }
 
         var escaped = string
+        // escape escape
+        escaped = escaped.replacingOccurrences(of: "\\", with: "\\\\")
```

Comment:
Won’t this now add more and more escaping characters every time a project is written?

Reply:
No, it wont. You can try it by adding build setting in Xcode with value \. It will become "\\" in proj file: 
https://github.com/RepositoryFork/xcproj/blob/d8a6995c23f8a5bd2f0aea68cb6461e7f08ee788/Fixtures/iOS/BuildSettings.xcodeproj/project.pbxproj#L103